### PR TITLE
Release sdk b022

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/workflow/creating-a-workflow-with-windows-powershell-activities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-with-windows-powershell-activities.md) | :warning:Warning | | [Details](#user-content-developer%2Fworkflow%2Fcreating-a-workflow-with-windows-powershell-activities.md)
[developer/prog-guide/stopprocesssample04-code-samples.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-code-samples.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/stopprocesssample04-csharp-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-csharp-sample-code.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/stopprocesssample04-vb-net-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-vb-net-sample-code.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-concepts.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-concepts.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-programmer-s-guide.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-programmer-s-guide.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-sample-code.md) | :white_check_mark:Succeeded | |
[developer/provider/accessdbprovidersample01.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/provider/accessdbprovidersample01.md) | :white_check_mark:Succeeded | |
[developer/provider/accessdbprovidersample02.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/provider/accessdbprovidersample02.md) | :white_check_mark:Succeeded | |
[developer/webservices/creating-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/creating-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/deploying-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/deploying-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/implementing-custom-authorization-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/implementing-custom-authorization-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/implementing-sessionconfiguration-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/implementing-sessionconfiguration-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/invoking-windows-powershell-cmdlets-with-management-odata-web-services.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/invoking-windows-powershell-cmdlets-with-management-odata-web-services.md) | :white_check_mark:Succeeded | |
[developer/webservices/management-odata-iis-extension-schema-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/management-odata-iis-extension-schema-files.md) | :white_check_mark:Succeeded | |
[developer/webservices/public-resource-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/public-resource-schema.md) | :white_check_mark:Succeeded | |
[developer/webservices/resource-mapping-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/resource-mapping-schema.md) | :white_check_mark:Succeeded | |
[developer/webservices/role-based-authorization-configuration-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/role-based-authorization-configuration-schema.md) | :white_check_mark:Succeeded | |
[developer/workflow/adding-windows-powershell-activities-to-the-visual-studio-toolbox.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/adding-windows-powershell-activities-to-the-visual-studio-toolbox.md) | :white_check_mark:Succeeded | |
[developer/workflow/common-workflow-parameters.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/common-workflow-parameters.md) | :white_check_mark:Succeeded | |
[developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md) | :white_check_mark:Succeeded | |
[developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md) | :white_check_mark:Succeeded | |
[developer/workflow/extending-the-windows-powershell-workflow-activity-controller.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/extending-the-windows-powershell-workflow-activity-controller.md) | :white_check_mark:Succeeded | |
[developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md) | :white_check_mark:Succeeded | |
[developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md) | :white_check_mark:Succeeded | |
[developer/workflow/writing-a-windows-powershell-workflow.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/writing-a-windows-powershell-workflow.md) | :white_check_mark:Succeeded | |

<a name="developer%2Fworkflow%2Fcreating-a-workflow-with-windows-powershell-activities.md"></a>
### [developer/workflow/creating-a-workflow-with-windows-powershell-activities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-with-windows-powershell-activities.md)
 - **Line 74**: **[Warning]** Invalid file link:(~/media/joindomainworkflow.png).
 - **Line 75**: **[Warning]** Invalid file link:(~/media/joindomainworkflow.png).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081554390359-2537%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rrQbXbk1ectaClZ9PKJ3lJLwzgYh9N1xtEgGmm9zIB8%3D&st=2018-06-08T15%3A59%3A09Z&se=2018-07-09T16%3A04%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: developer/webservices/public-resource-schema.md [hunk 1]
```diff
@@ -1,77 +1,83 @@
...
+Class DairyProduct : Product {
+
+Uint16 PercentFat;
+};
+Class POPProduct : Product {
+
+Boolean IsCarbonated;
+};
+
+```
```
Comment (sdwheeler) [inline developer/webservices/public-resource-schema.md]: Control character

File: developer/webservices/role-based-authorization-configuration-schema.md [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
+               </xs:complexType>
+           </xs:element>
+       </xs:all>
+       <xs:attribute name="Name" type="xs:string" use="required"/>
+       <xs:attribute name="DomainName" type="xs:string" use="optional"/>
+       <xs:attribute name="AuthenticationType" type="xs:string" use="required"/>
+       <xs:attribute name="GroupName" type="xs:string" use="required"/>
+   </xs:complexType>
+</xs:schema>
+```
```
Comment (sdwheeler) [inline developer/webservices/role-based-authorization-configuration-schema.md]: Control character

File: developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md [hunk 3]
```diff
@@ -1,118 +1,122 @@
...
+            ActivityCode = ActivityGenerator.GenerateFromCommandInfo(MyCommand, "MyNamespace");
+
+           CodeFile.Write(ActivityCode);
+            Console.ReadLine();
+
+        }
+    }
+}
+
+```
```
Comment (sdwheeler) [inline developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md]: Control character

File: developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md [hunk 4]
```diff
@@ -1,137 +1,140 @@
...
+                {
+                    Console.WriteLine(v.Message);
+                }
+
+            runspace.Close();
+        }
+    }
+}
+
+```
```
Comment (sdwheeler) [inline developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md]: Control character

File: developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md [hunk 5]
```diff
@@ -1,244 +1,251 @@
...
+                if (registrationSucceeded)
+                {
+                    schedJobDefinition.Remove(true);
+                }
+            }
+        }
+    }
+}
+
+```
```
Comment (sdwheeler) [inline developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md]: Control character

File: developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md [hunk 6]
```diff
@@ -1,44 +1,48 @@
...
+        sequence {
+        Get-WmiObject -PSComputerName $Computer -PSCredential $MachineCred
+        Add-Computer -PSComputerName $Computer -PSCredential $DomainCred
+        Restart-Computer -ComputerName $Computer -Credential $MachineCred -For PowerShell -Force -Wait -PSComputerName ""
+        Get-WmiObject -PSComputerName $Computer -PSCredential $MachineCred
+        }
+    }
+ }
+
+```
```
Comment (sdwheeler) [inline developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md]: Control character

File: developer/webservices/resource-mapping-schema.md [hunk 7]
```diff
@@ -1,199 +1,202 @@
...
+    </xs:all>
+  </xs:complexType>
+  <xs:complexType name="GetAssociationReferenceType">
+    <xs:all>
+      <xs:element name="Cmdlet" type="xs:string" />
+      <xs:element name="ParameterForThisObject" type="xs:string" />
+    </xs:all>
+  </xs:complexType>
+</xs:schema>
+```
```
Comment (sdwheeler) [inline developer/webservices/resource-mapping-schema.md]: Control character

File: developer/webservices/public-resource-schema.md (hunk 1 above)
Comment (bobbytreed) [reply developer/webservices/public-resource-schema.md]: Fixed This

File: developer/webservices/role-based-authorization-configuration-schema.md (hunk 2 above)
Comment (bobbytreed) [reply developer/webservices/role-based-authorization-configuration-schema.md]: Fixed This

File: developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md (hunk 4 above)
Comment (bobbytreed) [reply developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md]: Fixed This

File: developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md (hunk 6 above)
Comment (bobbytreed) [reply developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md]: Fixed This

File: developer/webservices/resource-mapping-schema.md (hunk 7 above)
Comment (bobbytreed) [reply developer/webservices/resource-mapping-schema.md]: Fixed This

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/workflow/creating-a-workflow-with-windows-powershell-activities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-with-windows-powershell-activities.md) | :warning:Warning | | [Details](#user-content-developer%2Fworkflow%2Fcreating-a-workflow-with-windows-powershell-activities.md)
[developer/prog-guide/stopprocesssample04-code-samples.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-code-samples.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/stopprocesssample04-csharp-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-csharp-sample-code.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/stopprocesssample04-vb-net-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/stopprocesssample04-vb-net-sample-code.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-concepts.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-concepts.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-programmer-s-guide.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-programmer-s-guide.md) | :white_check_mark:Succeeded | |
[developer/prog-guide/windows-powershell-sample-code.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/prog-guide/windows-powershell-sample-code.md) | :white_check_mark:Succeeded | |
[developer/provider/accessdbprovidersample01.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/provider/accessdbprovidersample01.md) | :white_check_mark:Succeeded | |
[developer/provider/accessdbprovidersample02.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/provider/accessdbprovidersample02.md) | :white_check_mark:Succeeded | |
[developer/webservices/creating-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/creating-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/deploying-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/deploying-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/implementing-custom-authorization-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/implementing-custom-authorization-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/implementing-sessionconfiguration-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/implementing-sessionconfiguration-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/invoking-windows-powershell-cmdlets-with-management-odata-web-services.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/invoking-windows-powershell-cmdlets-with-management-odata-web-services.md) | :white_check_mark:Succeeded | |
[developer/webservices/management-odata-iis-extension-schema-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/management-odata-iis-extension-schema-files.md) | :white_check_mark:Succeeded | |
[developer/webservices/public-resource-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/public-resource-schema.md) | :white_check_mark:Succeeded | |
[developer/webservices/resource-mapping-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/resource-mapping-schema.md) | :white_check_mark:Succeeded | |
[developer/webservices/role-based-authorization-configuration-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/webservices/role-based-authorization-configuration-schema.md) | :white_check_mark:Succeeded | |
[developer/workflow/adding-windows-powershell-activities-to-the-visual-studio-toolbox.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/adding-windows-powershell-activities-to-the-visual-studio-toolbox.md) | :white_check_mark:Succeeded | |
[developer/workflow/common-workflow-parameters.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/common-workflow-parameters.md) | :white_check_mark:Succeeded | |
[developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-activity-from-a-windows-powershell-cmdlet.md) | :white_check_mark:Succeeded | |
[developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-by-using-a-windows-powershell-script.md) | :white_check_mark:Succeeded | |
[developer/workflow/extending-the-windows-powershell-workflow-activity-controller.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/extending-the-windows-powershell-workflow-activity-controller.md) | :white_check_mark:Succeeded | |
[developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md) | :white_check_mark:Succeeded | |
[developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/scheduling-jobs-with-the-windows-powershell-api.md) | :white_check_mark:Succeeded | |
[developer/workflow/writing-a-windows-powershell-workflow.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/writing-a-windows-powershell-workflow.md) | :white_check_mark:Succeeded | |

<a name="developer%2Fworkflow%2Fcreating-a-workflow-with-windows-powershell-activities.md"></a>
### [developer/workflow/creating-a-workflow-with-windows-powershell-activities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b022/developer/workflow/creating-a-workflow-with-windows-powershell-activities.md)
 - **Line 74**: **[Warning]** Invalid file link:(~/media/joindomainworkflow.png).
 - **Line 75**: **[Warning]** Invalid file link:(~/media/joindomainworkflow.png).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081742190388-2537%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3K6MFn8W3oTlOJ8Bi9MR1IeIe0H1jmuzjl4no3RG68w%3D&st=2018-06-08T17%3A47%3A05Z&se=2018-07-09T17%3A52%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**